downloads/swan-lake-release-notes/swan-lake-alpha/RELEASE_NOTE.md

```diff
@@ -0,0 +1,1432 @@
+        - [Support for Empty XML Values](#support-for-empty-xml-values)
+        - [Improved Usage of Listener Declarations with Listener Types](#improved-usage-of-listener-declarations-with-listener-types)
+        - [Improvements to the `match` Statement](#improvements-to-the-match-statement)
+        - [Support for Cyclic Union Types](#support-for-cyclic-union-types)
+        - [Updated Syntax for User-Defined Error Constructions](#updated-syntax-for-user-defined-error-constructions)
+        - [Changes on Casting with Errors](#changes-on-casting-with-errors)
+        - [Changes on `toString` and `toBalString` with Errors](#changes-on-toString-and-toBalString-with-errors)
+        - [Changes on Object Type Inclusion with Qualifiers](#changes-on-object-type-inclusion-with-qualifiers)
+        - [Changes on Record Type Inclusion with Rest Descriptors](#changes-on-record-type-inclusion-with-rest-descriptors)
+        - [Improved Listener Declaration](#improved-listener-declaration)
+        - [Referring Lang Library Modules Without Using Quoted Identifiers](#referring-lang-library-modules-without-using-quoted-identifiers)
+        - [Improved Lang Library Methods](#improved-lang-library-methods)
+    - [Runtime](#runtime)
+        - [Configurable](#configurable)
+    - [Developer Tools](#developer-tools)
+        - [Language Server](#language-server)
+        - [Project API](#project-api)
+        - [Debugger](#debugger)
+        - [Test Framework](#test-framework)
+        - [Bindgen Tool](#bindgen-tool)
+        - [Maven Resolver](#maven-resolver)
+        - [Ballerina Shell REPL [EXPERIMENTAL]](#ballerina-shell-repl-experimental)
+        - [Documentation](#documentation)
+    - [Standard Library](#standard-library)
+        - [HTTP Module Improvements](#http-module-improvements)
+        - [WebSocket Module Improvements](#websocket-module-improvements)
+        - [gRPC Module Improvements](#grpc-module-improvements)
+        - [Security Improvements](#security-improvements)
+        - [GraphQL Module Improvements](#graphql-module-improvements)
+        - [Common Changes in Messaging Modules](#common-changes-in-messaging-modules)
+        - [Kafka Module Improvements](#kafka-module-improvements)
+        - [NATS Module Improvements](#nats-module-improvements)
+        - [NATS Streaming Module Improvements](#nats-streaming-module-improvements)
+        - [RabbitMQ Streaming Module Improvements](#rabbitmq-streaming-module-improvements)
+        - [Time Module Improvements](#time-module-improvements)
+        - [Rename System Module to OS](#rename-system-module-to-os)
+        - [Runtime Module Improvements](#runtime-module-improvements)
+        - [Email Module Improvements](#email-module-improvements)
+        - [Introduced Modules](#introduced-modules)
+        - [Removed Modules](#removed-modules)
+    - [Code to Cloud](#code-to-cloud)
+    - [Breaking Changes](#breaking-changes)
```

Comment:
Shouldn't they be emphasized as "breaking changes"? @rdhananjaya

Reply:
They should. What I meant was moving it with the "Breaking Changes" heading.

Reply:
I went through old release notes; we seem to abandon the breaking changes section in Swan Lake release notes

Reply:
Depends, right? If we have breaking changes that cannot be classified directly under "What's new" we have a "Breaking Changes" section.

For example, we seem to have had one for slp7.

Reply:
I didn't see the slp7 note, so in that case we can just remove the `language` heading and move the content under breaking change?

Reply:
Yeah, can do that too. Was wondering because there are breaking changes in the stdlib too, but they probably need to be highlighted so need their own headings.

+1 keep the breaking changes section as is and remove the just the "Language" sub-heading.

Reply:
Fixed. Thanks!